src/utils/cloudinary.js

```diff
@@ -101,13 +101,13 @@ const isKeyInTransformation = (transformation, key) => {
 
 const addTextTracks = (tracks, videojs) => {
   tracks.forEach(track => {
-    if (track.src) {
+    if (track.src && track.src.endsWith('.vtt')) {
       fetch(track.src, GET_ERROR_DEFAULT_REQUEST).then(r => {
         if (r.status >= 200 && r.status <= 399) {
           videojs.addRemoteTextTrack(track, true);
         }
       });
-    } else if (videojs.pacedTranscript) {
+    } else if (videojs.pacedTranscript && (!track.src || track.src.endsWith('.transcript'))) {
```

Comment:
how does it work? i do not see any mention of url property is it being translated somewhere to src?

Reply:
yes, our original API was using `url` for VTTs, way before we added transcript support
https://github.com/cloudinary/cloudinary-video-player/blob/edge/src/video-player.js#L225